workers/unity/Assets/Gdk/TestUtils/Tests/Editmode/HybridGdkSystemTests.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.TestUtils.EditmodeTests
+{
+    public class HybridSystemTests
+    {
+        private struct TestPreparation : IComponentData
+        {
+            public int Value;
+        }
+
+        [UsedImplicitly]
+        [DisableAutoCreation]
```

Comment:
Do we need this? If a world is including all systems that exist then it will add all systems we've currently written. If this is needed here the presumably we need it everywhere.

Reply:
Not necessary for this one since it's in editor tests assembly.

I do wonder though if it may be necessary to do for all others, but that can be decided later.

I'll remove it from this one